### PR TITLE
Version 2

### LINKED ISSUE #13: Move colors to o-colors

### LINKED ISSUE #56: mixin overlap
When using mixins exclusively, calling `@include oTeaser(('default', 'images'), ('hero'))` doesn't bring in all of the `hero` styling — specifically, it doesn't seem include any styling that is supported by `large`, unless `large` itself is explicitly included.

I haven't tried it with any other themes/elements, though, so could be an exception.

### PR DESCRIPTION
**Pull request by onishiweb**
Updates to new o-colors and o-typography fixing the colors to the new palette and organising fonts and vertical spacing to the new rules in o-typography.

Also closes #56 
Closes #13

### CODE REVIEW

File: src/scss/_color-use-cases.scss [hunk 1]
```diff
@@ -1,25 +1,42 @@
...
+@include oColorsSetUseCase(o-teaser-package-basic, border, 'claret-70');
+@include oColorsSetUseCase(o-teaser-package-basic-hover, text, 'claret-30');
+@include oColorsSetUseCase(o-teaser-package-special-report, background, 'claret-70');
+
 // Timestamp - Live blog
-@include oColorsSetColor(live-blog-closed, oColorsGetPaletteColor('grey-tint4'));
-@include oColorsSetColor(live-blog-coming-soon, #fd9d00);
-@include oColorsSetColor(live-blog-in-progress, oColorsGetPaletteColor('red'));
+@include oColorsSetColor(live-blog-closed, oColorsGetPaletteColor('black-60'));
+@include oColorsSetColor(live-blog-coming-soon, #fd9d00); // ??
```
Comment (JakeChampion) [inline src/scss/_color-use-cases.scss]: Have you asked a designer about what it should be?

File: src/scss/_variables.scss [hunk 2]
```diff
@@ -3,18 +3,17 @@ $o-teaser-is-silent: true !default;
 
 // Colors not in o-colors
 // TODO: Migrate these to `o-colors`
-$_o-teaser-color-border: mix(#000000, oColorsGetPaletteColor('pink'), 20%); // #D1C6B7;
-$_o-teaser-color-hero-background: #f3dec8;
-$_o-teaser-color-hero-opinion-background: #275c85;
-$_o-teaser-color-opinion-tag: #275c85;
+// $_o-teaser-color-border: mix(#000000, oColorsGetPaletteColor('pink'), 20%); // #D1C6B7;
```
Comment (JakeChampion) [inline src/scss/_variables.scss]: I think we should remove commented-out code

File: src/scss/elements/_default.scss [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
 /// sets the font and basic display properties
 @mixin oTeaserBase {
-	@include oTypographyProgressiveFont(sans, m);
+	@include oTypographySans(-1);
+	@include oTypographyMargin($bottom: 4);
 	position: relative;
-	margin-bottom: 15px;
-	text-rendering: optimizeLegibility;
 	width: 100%;
+	text-rendering: optimizeLegibility;
```
Comment (JakeChampion) [inline src/scss/elements/_default.scss:8]: Interesting that this isn't included by o-normalise or o-typography

File: src/scss/_color-use-cases.scss (hunk 1 above)
Comment (onishiweb) [reply src/scss/_color-use-cases.scss]: Not this time, as these were already not in the palette I think it's ok to leave them to be updated at a later date.

File: src/scss/elements/_default.scss (hunk 3 above)
Comment (onishiweb) [reply src/scss/elements/_default.scss:8]: It is in `o-normalise` if you use the CSS reset part of it, but can't be sure everyone is using that at the moment.

File: README.md [hunk 4]
```diff
@@ -152,6 +152,10 @@ Uses the `o-teaser--video` modifier.
 
 ## Migration guide
 
+### Migrating from v1 to v2
+
+V1 -> V2 introduces the new major of o-colors and o-typography. Updating to this new version will mean updating any other components that you have which are using o-colors and o-typography. There are no other breaking changes in this release.
```
Comment (JakeChampion) [inline README.md]: And the new o-labels

**Review by JakeChampion — APPROVED** [review]

**Review by rowanmanning — APPROVED** [review]

File: src/scss/_color-use-cases.scss (hunk 1 above)
Comment (alicebartlett) [reply src/scss/_color-use-cases.scss]: Can we add this as an inline TODO in that case

**Comment by alicebartlett** [comment]
There are *a lot* of magic numbers in here- in particular padding. It would be better if that had been commented eg 
`padding-top: 20px; // magic number to line up with button`
o-typography has some helper sass mixins for getting vertical rhythm right which might also be useful here, but it would be a bit of a fiddle to get it in.
I think we should merge this anyway, just something I'd like us to be more aware of in the future.

**Pull request merged**